Dockerfile.ubuntu20

```diff
@@ -28,14 +29,14 @@ RUN apt-get -qq update \
 #         mlnx-ofed-hpc-user-only
 
 # IB perftest with GDR
-ENV PERFTEST_VERSION=4.5-0.20
-ENV PERFTEST_VERSION_HASH=gac7cca5
+# ENV PERFTEST_VERSION=4.5-0.20
```

Comment:
These are used for the old perftest repo, and are now stale, but we should probably replace the commithash with a new one in your repo.